src/init_files/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Rename to `map_settings.json` or something more meaningful.
This is not really [a Pythonic config](https://docs.python.org/3/library/configparser.html).